cli-ext/src/services/actions-codegen/template.js

```diff
@@ -1,8 +1,3 @@
-const { buildSchema, printSchema, parse } = require('graphql');
```

Comment:
These removals should be undone because these dependencies need to be in scope when we eval a the codegen script.